### PR TITLE
dwebp: add page

### PR DESCRIPTION
**Pull request by markthequark**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

-o flag has no spelled-out version like --output

### CODE REVIEW

File: pages/common/dwebp.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# dwebp
+
+> dwebp decompresses WebP files into PNG, PAM, PPM or PGM images.
```
Comment (marchersimon) [inline pages/common/dwebp.md:3]:
```suggestion
> `dwebp` decompresses WebP files into PNG, PAM, PPM or PGM images.
```

File: pages/common/dwebp.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+> Animated WebP files are not supported.
+> More information: <https://developers.google.com/speed/webp/docs/dwebp/>.
+
+- convert webp to png
```
Comment (marchersimon) [inline pages/common/dwebp.md:7]:
```suggestion
- Convert a `webp` file to a `png` file:
```

File: pages/common/dwebp.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+`dwebp input.webp -o output.png`
```
Comment (marchersimon) [inline pages/common/dwebp.md:9]:
```suggestion
`dwebp {{path/to/input.webp}} -o {{path/to/output.png}}`
```

**Review by marchersimon — COMMENTED** [review]
Just a few minor suggestion, but looks good overall. Would you maybe like to add more options like `-flip`, `-scale` or `-mt`?
We could also add an example for other filetypes like this:
```
- Convert a `webp` file into a specific filetype:

`dwebp {{path/to/input.webp}} -bmp|-tiff|-pam|-ppm|-pgm|-yuv -o {{path/to/output}}`
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @markthequark

**Pull request merged**